### PR TITLE
Add missing SNS API calls

### PR DESCRIPTION
**Pull request by asgoel**
Adds the missing SNS API calls per #267. cc @jnicklas

### CODE REVIEW

File: lib/ex_aws/sns/parsers.ex [hunk 1]
```diff
@@ -108,6 +139,39 @@ if Code.ensure_loaded?(SweetXml) do
                           subscription_arn: ~x"./SubscriptionArn/text()"s,
                           topic_arn: ~x"./TopicArn/text()"s,
                         ],
+                        next_token: ~x"./ListSubscriptionsResult/NextToken/text()"s,
```
Comment (asgoel) [inline lib/ex_aws/sns/parsers.ex]: @jnicklas I added this in because it looks like it was missing. I added a separate test for it as well.

**Comment by benwilson512** [comment]
This looks fine to me, is it ready to go as far as features you wanted to add?

**Comment by asgoel** [comment]
@benwilson yes

**Comment by asgoel** [comment]
@benwilson512 sorry - meant to say that this one is done, and you can go ahead and merge if it's ok.

**Pull request merged**